setup.py

```diff
@@ -59,6 +59,12 @@ def path_exists(filename):
     "Type your Channel's name, then press `ENTER`. (You can change this later) "
 )
 print("\n--------------------------")
+print("\nNineth, what's your Owner ID? (example: 1234,2345,1234)")
```

Comment:
I would add how to get your Twitch ID

You can get it from clicking on a profile in Chatterino
Some bots (like Oura_Bot’s !whois command) let you see the ID.